### PR TITLE
Regenerate for Microsoft.Resources version 2019-07-01

### PR DESCRIPTION
**Pull request by shenglol**
Regenerate to add support for deployment what-if operations at subscription and resource group scope.

Corresponding Swagger PR: Azure/azure-rest-api-specs#7254.

### CODE REVIEW

**Review by markcowl — CHANGES_REQUESTED** [review]
- Please add a description
- Please include the generation metadate file created by generate.ps1 here: `eng\mgmt\mgmtmetadata\<rpName>.txt`
- Please include a link to the rest-api-specs change corresponding to thsi change.

**Comment by shenglol** [comment]
@markcowl Would you mind taking another look at the PR? I've made the changes. Thanks!

File: SDKs/_metadata/resources_resource-manager.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: latest
```
Comment (markcowl) [inline SDKs/_metadata/resources_resource-manager.txt]: this is not the proper place for this file.  It should be in: eng\mgmt\mgmtmetadata\<rpName>.txt

**Review by markcowl — CHANGES_REQUESTED** [review]

File: SDKs/_metadata/resources_resource-manager.txt (hunk 1 above)
Comment (shenglol) [reply SDKs/_metadata/resources_resource-manager.txt]: Sorry I got a bit confused with the auto generated path. Moved it to the right location.

**Review by markcowl — APPROVED** [review]

**Pull request merged**